signal/plotting.py

```diff
@@ -1,33 +1,27 @@
 import matplotlib.pyplot as plt
 import csv
 import os
+import argparse
 
 
-def plot_data(filename_data, args):
+def plot_data(filename_data, used_rl):
     iterations = []
     losses = []
     hinge_losses = []
     rl_losses = []
     entropies = []
     accuracies = []
 
-    dir_path = os.path.dirname(os.path.realpath(__file__))
-    dir_path = dir_path.replace("/baseline", "")
-    full_filename_data = os.path.join(dir_path, filename_data)
+    plot_path = filename_data.replace('.csv', '_plot.png')
```

Comment:
Minor suggestion:

you could do something like https://stackoverflow.com/questions/541390/extracting-extension-from-filename-in-python
It's probably the intended tool for this type of string manipulation.